### DISCUSSION TITLE
Q: unpkg.com 访问css文件 520

### DISCUSSION THREAD

**Discussion by aaaaaaanthony** [walinejs/waline#2426] [Q&A] [answered]
### Describe the bug | 问题描述

<img width="692" alt="image" src="https://github.com/walinejs/waline/assets/30170561/c6b4d938-505f-44a9-a096-410630097dc6">

返回520 https://unpkg.com/@waline/client@v3/dist/waline.css
返回200 https://unpkg.com/@waline/client@3.1.3/dist/waline.css

### Website URL | 问题网站

https://waline.runtimes.cc/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

MongoDB

**Comment by Mister-Hope** [comment, marked as answer]
In my network, the above link returns a correct 302 to the below link

![image](https://github.com/walinejs/waline/assets/33315834/4128fe1d-228b-4477-b5ca-6216099cdcfe)

Also I am expecting this should be redirected to a link with new version once new version lands.
